clientd3d/d3drender.c

```diff
@@ -630,15 +636,15 @@ void D3DRenderShutDown(void)
 	}
 }
 
-void D3DRenderBegin(room_type *room, Draw3DParams *params)
+void D3DRenderBegin(room_type* room, Draw3DParams* params)

```

Comment:
It's super hard to review this with all of the reformatting changes.  Can you put them back?

Reply:
Hiding whitespace changes under the gear up top helps quite a bit.
![image](https://github.com/Meridian59/Meridian59/assets/467443/f4cecbbf-58f7-4104-a73b-6ac1d90cf064)